### DISCUSSION TITLE
Broken link

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3530] [General] [answered]
Hello, i just wanted to let you know that there is a broken link on the jbrowse 2 website (Configure JBrowse using the CLI):
https://jbrowse.org/jb2/docs/tutorials/config_cli/here/

Thanks

**Comment by cmdcolin** [comment, marked as answer]
Thanks for reporting! I will try and add a broken link checker to our CI, I think it will help catch errors like this in the future

**Reply by cmdcolin** [reply]
and fix that particular error :)

**Reply by cmdcolin** [reply]
should be fixed after this update, should be online shortly https://github.com/GMOD/jbrowse-components/pull/3532

**Reply by MehdiiYac** [reply]
Thank you!
